### PR TITLE
Ft  basic UI

### PR DESCRIPTION
**Pull request by Bateyjosue**
# FEATURE DESCRIPTION
## TASK:
In this project, I create a form of UI for our school library app. This way it can be invoked as an executable and not something you use in IRB exclusively.
ScreenShot

### Cheers

# Happy Review

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -1,9 +1,158 @@
...
+
+  puts 'Date [YYYY-MM-DD] :'
+  date = gets.chomp
+
+  rentals.push(Rental.new(date, @persons[person_index], @books[book_index]))
+
+  puts "Rental created successfully\n"
+end
+
+def list_rental()
```
Comment (SajjadAhmad14) [inline main.rb]: There is an error when the user tries to list all rentals. Kindly see the screenshot below.

rent

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Josh, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
README is professional. :heavy_check_mark:
Git flow is followed. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: main.rb (hunk 1 above)
Comment (Bateyjosue) [reply main.rb]: Oh okay. Thanks, Bro. Thanks, I messed up when fixing linters... 
Gotcha

Let me work on it...

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**